src/Lykke.Service.PaySettlement.Core/Domain/ITradeOrdersRepository.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Lykke.Service.PaySettlement.Core.Domain
+{
+    public interface ITradeOrdersRepository
+    {
+        Task InsertOrMergeTradeOrderAsync(ITradeOrder tradeOrder);
+        Task<IEnumerable<ITradeOrder>> GetAsync();
+        Task<IEnumerable<ITradeOrder>> GetAsync(string assetPair);
+        Task DeleteAsync(ITradeOrder tradeOrder);
+        Task DeleteAsync(IEnumerable<ITradeOrder> tradeOrders);
```

Comment:
This method is not used. Please take attention to Resharper notifications.

Reply:
This is added for future usage. I will remove it so far.